assets/global.js

```diff
@@ -1359,3 +1358,50 @@ class AccountIcon extends HTMLElement {
 }
 
 customElements.define('account-icon', AccountIcon);
+
+class ScrollButton extends HTMLElement {
+  constructor() {
+    super();
+
+    this.icon = this.querySelector('.scroll-button--link');
```

Comment:
I don't recommend using default CSS selectors as anchors. 
Better use data-attributes (it's my favourite), or create special js classes.
My recommendation is:
```suggestion
    this.icon = this.querySelector('[data-scroll-button-link]');
```